IPython/html/static/widgets/js/types.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) IPython Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+define([
+    "base/js/utils"
+], function(utils){
+
+    return {
+        widget_serialization: {
```

Comment:
I'm deleting list_of_numbers...

but I'll rename so the name is shorter (since it gets transferred across *every* *single* *time* a list of children is transferred.